bumble/device.py

```diff
@@ -2067,13 +2067,13 @@ def on_authentication_user_confirmation_request(self, connection, code):
         # Ask what the pairing config should be for this connection
         pairing_config = self.pairing_config_factory(connection)
 
-        can_confirm = pairing_config.delegate.io_capability not in {
+        can_compare = pairing_config.delegate.io_capability not in {
             smp.SMP_NO_INPUT_NO_OUTPUT_IO_CAPABILITY,
             smp.SMP_DISPLAY_ONLY_IO_CAPABILITY
         }
 
         # Respond
-        if can_confirm and pairing_config.delegate:
+        if can_compare and pairing_config.delegate:
```

Comment:
I think the `and pairing_config.delegate` here is a leftover from a previous version and isn't needed (it can't be None anymore, and in fact we access `pairing_config.delegate.io_capability` just above). The same applies to line 2116.

Reply:
Done :+1: